src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,142 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    private static class Node<T> {
+        T item;
+        Node<T> next;
+        Node<T> prev;
+
+        public Node(T value, Node<T> next, Node<T> prev) {
+            this.item = value;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
+    public MyLinkedList() {
+        first = null;
+        last = null;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        Node<T> newNode;
+        if (isEmpty()) {
+            newNode = new Node<>(value, null, null);
+            first = newNode;
+            last = newNode;
+        } else {
+            newNode = new Node<>(value, null, last);
```

Comment:
you may move this line before if else, that will make code shorter

Reply:
done